chapter_3/embedded_ansible.md

```diff
@@ -1,9 +1,77 @@
 # Embedded Ansible
 
-- Enabling Server Role
-- rake evm:status
-- evm.log and automation.log
-- Add GiHub playbook Repository
-- Ansible modules for OpenStack (http://docs.ansible.com/ansible/latest/list_of_cloud_modules.html#openstack)
+In ManageIQ after release Fine, Ansible is embedded within ManageIQ itself. Before Fine release, automation was supported in ManageIQ but we had to write a Ruby script which uses custom domain.
 
-Issue faced while enabling Embedded Ansible Server Role
+Since Fine release automation has been easier in way that we just need to write Ansible playbooks. Embedded Ansible provides integrity to run Playbooks.
+
+After the integration of OpenStack provider with ManageIQ, It is the time to enable *EmbeddedAnsible* Server Role in Server Controls of ManageIQ.
+
+1. Navigate to **Administrator|EVM -> Configuration**. This will land you on the screen of *Server Settings*.
+
+2. Enable the EmbeddedAnsible server role.
+
+3. Click the Save button to carry forward the changes.
+
+![Fig 1-EmbeddedAnsible Role](../images/chapter3/EmbeddedAnsible_Role.png "EmbeddedAnsible Role")
+
+| Note |ManageIQ will create events about EmbeddedAnsible activation. This will take some time upto 5 - 10 mins.|
+|------|:------|
+
+You can check the status of the task in ManageIQ by navigating to **Administrator|EVM -> Tasks -> All Tasks tab** .
+You can also check that whether the EmbeddedAnsible role has started or not by `rake evm:status`.
+
+| Warning |If the Server Role is not started, do check the `evm.log` and `automation.log` files.|
```

Comment:
Logs are located in `log/evm.log` and `log/automation.log` after we execute `vmdb`